HACKING.md

```diff
@@ -23,6 +23,86 @@ TODO: Explain how to modify initrd when there is Secure boot / FDE.
   - snapcraft
   - etc
 
+# Testing with spread
+
+## Prerequisites for testing
+
+You need to have the following software installed before you can test with spread
+ - Go (https://golang.org/doc/install or ```sudo snap install go```)
+ - Spread (install from source as per below)
+
+## Installing spread
+
+You can install spread by simply using ```snap install spread```, however this does not allow for the lxd-backend to be used.
+To use the lxd backend you need to install spread from source, as the LXD profile support has not been upstreamed yet.
+This document will be updated with the upstream version when this happens. To install spread from source you need to do the following.
+
+```
+git clone https://github.com/Meulengracht/spread
```

Comment:
Could we maybe comment that we need to change the document when `https://github.com/snapcore/spread/pull/136` is merged?

Reply:
Yes, definiately a good suggestion